csrc/moe/marlin_moe_wna16/marlin_template.h

```diff
@@ -774,8 +648,8 @@ __global__ void Marlin(
                 (threadIdx.x % b_sh_stride_threads) * b_thread_vecs;
   b_gl_rd += b_sh_stride * slice_col;
   b_gl_rd += b_gl_rd_delta_o * slice_row;
-  int b_sh_wr = threadIdx.x * b_thread_vecs;
-  int b_sh_rd = threadIdx.x * b_thread_vecs;
+  auto b_sh_wr = threadIdx.x * b_thread_vecs;
+  auto b_sh_rd = threadIdx.x * b_thread_vecs;
```

Comment:
Why did you move to using auto in many places?

Reply:
Just follow this pr https://github.com/vllm-project/vllm/pull/15160